api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -388,23 +387,33 @@ public void sendFileLengthsEgressRemediationEmail(DbUser dbUser, EgressRemediati
   }
 
   @Override
-  public void sendPublishWorkspaceByAdminEmail(
+  public void sendPublishWorkspaceEmail(
```

Comment:
Support has requested that we do not send an email notification when an admin publishes a workspace, as these are managed by the DRC. I have additional questions about this process, so for now, I am combining the two publish methods (by Admin and by Owner).

Depending on the answers I receive, I'll create a new PR to remove the email call from the admin service.